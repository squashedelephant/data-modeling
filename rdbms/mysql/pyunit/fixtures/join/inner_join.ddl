CREATE TABLE user(id INT NOT NULL AUTO_INCREMENT, name CHAR(15) NOT NULL, PRIMARY KEY (id));
CREATE TABLE profile(id INT NOT NULL AUTO_INCREMENT, user_id INT, password VARCHAR(30), email VARCHAR(50), PRIMARY KEY (id), INDEX user_idx (user_id), FOREIGN KEY (user_id) REFERENCES user(id) ON DELETE CASCADE);
