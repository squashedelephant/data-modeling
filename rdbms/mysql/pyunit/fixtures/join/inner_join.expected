Bill Gates
Microsoft
bill_gates@microsoft.com
Steve Jobs
Apple
steve_jobs@apple.com
Larry Ellison
Oracle
larry_ellison@oracle.com
